content/yaml/yaml-conditional-run.md

```diff
@@ -75,21 +65,24 @@ The `condition` you specify will be evaluated during the build. The build will b
 
 The current environment is accessible under the `env` variable.
```

Comment:
```suggestion
Information about the current environment is accessible under the `env` variable.
```

Reply:
It would be nice to briefly mention the use case, e.g. so you can check branch information in the condition.